src/python/pants/backend/python/packaging/pyoxidizer/config.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from string import Template
+from textwrap import indent
+from typing import Optional
+
+DEFAULT_TEMPLATE = """
+def make_exe():
+    dist = default_python_distribution()
+    policy = dist.make_python_packaging_policy()
+
+    # Note: Adding this for pydanic and libs that have the "unable to load from memory" error
+    # https://github.com/indygreg/PyOxidizer/issues/438
+    policy.resources_location_fallback = "filesystem-relative:lib"
+
+    python_config = dist.make_python_interpreter_config()
+    $ENTRY_POINT
+
+    exe = dist.to_python_executable(
+        name="$NAME",
+        packaging_policy=policy,
+        config=python_config,
+    )
+
+    # pip_download requires that wheels are available for each dep
+    # exe.add_python_resources(exe.pip_download($WHEELS))
+    exe.add_python_resources(exe.pip_install($WHEELS))
```

Comment:
We'll likely want to resolve the relevant wheels via PEX at some point (since that will apply a user's repository settings and etc), but fine as a TODO for later.

Reply:
I agree, there will be a lot of re-factoring to be done on this while in experimental :)